app/knights_config.py

```diff
@@ -0,0 +1,86 @@
+knights_configurations = {
+    "lancelot": {
+        "name": "Lancelot",
+        "power": 35,
+        "hp": 100,
+        "armour": [],
+        "weapon": {
+            "name": "Metal Sword",
+            "power": 50,

```

Comment:
There is a trailing comma after the 'power' key in the 'weapon' dictionary. This is not necessary and can be removed.